apps/backend/app/routers/mcp.py

```
@@ -0,0 +1,234 @@
+        return None
+    return await db.get_user_by_id(claims["sub"])
+
+
+# -- Main handler -------------------------------------------------------------
+
+@router.post("/mcp")
+async def mcp_handler(request: Request) -> JSONResponse:
+    """MCP Streamable HTTP endpoint (JSON-RPC 2.0)."""
+    try:
+        body = await request.json()
+    except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)